1.0/docs/devguide/local-dom.md

```diff
@@ -226,9 +226,9 @@ You can also locate nodes in the local DOM using `querySelector`, `querySelector
 
     var cancelButton = Polymer.dom(this.root).querySelector('#cancelButton');
 
-    // Shorthand for finding local DOM children by ID
+    // Shorthand for finding local DOM children by selector
     // (equivalent to the above):
-    this.$$('cancelButton');
+    this.$$('#cancelButton');
```

Comment:
i wonder if a class would be a better example. It's usually assumed only a single node on the page has an ID.